src/build.d

```diff
@@ -1176,7 +1176,7 @@ auto sourceFiles()
             parse.d parsetimevisitor.d permissivevisitor.d printast.d safe.d sapply.d scanelf.d scanmach.d
             scanmscoff.d scanomf.d semantic2.d semantic3.d sideeffect.d statement.d statement_rewrite_walker.d
             statementsem.d staticassert.d staticcond.d stmtstate.d target.d templateparamsem.d traits.d
-            transitivevisitor.d typesem.d typinf.d utils.d visitor.d vsoptions.d foreachvar.d
+            transitivevisitor.d typesem.d typinf.d utils.d visitor.d vsoptions.d foreachvar.d unused.d
```

Comment:
This is a confusing name because people will think the module is actually not used. 

```suggestion
            transitivevisitor.d typesem.d typinf.d utils.d visitor.d vsoptions.d foreachvar.d detect_unused.d
```

Reply:
Fixed.